Objects/asm_trampoline.S

```diff
@@ -0,0 +1,30 @@
+    .text
+    .globl	_Py_trampoline_func_start
+_Py_trampoline_func_start:
```

Comment:
A comment with equivalent C code would be nice. Here the assembly is mostly passing arguments and calling an indirect function on x86-64, I guess arm is the same?

Reply:
P.S I only know x86-64 assembly so no idea about arm :)